index.html

```diff
@@ -45,37 +45,41 @@ <h2>Our Partners</h2>
-              </div>
-              <div class="business-partners-display">
-              <img alt="" src="/images/1-6-Microsoft.svg" class="business-partners-logo">
-              </div>
-              <div class="business-partners-display">
-              <img alt="" src="/images/1-7-Duo .png" class="business-partners-logo">
-              </div>
-              <div class="business-partners-display">
-              <img alt="" src="/images/2-1-member.png" class="business-partners-logo">
-              </div>
-              <div class="business-partners-display">
-              <img alt="" src="/images/2-2-datacamp.svg" class="business-partners-logo">
-              </div>
-              
-              <div class="business-partners-display">
-              <img alt="" src="/images/2-3-Antwerp.png" class="business-partners-logo">
-              </div>
-        </section>
+            <img alt="PrintSimple logo" src="images/1-1-print-smple.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Open Knowledge Belgium logo" src="images/1-2-open%20-knowledge.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Digital Belgium logo" src="images/1-3-digital-belgium.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Artevelde Hoge School logo" src="images/1-4-artevelde.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Bruxelles Economie et Emploi logo" src="images/1-5-bruxelles.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Microsoft logo" src="images/1-6-Microsoft.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Duo for Job logo" src="images/1-7-Duo .png" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Member logo png" src="images/2-1-member.png" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Data Camp logo" src="images/2-2-datacamp.svg" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="UX Antwerp logo" src="images/2-3-Antwerp.png" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Open Recognition logo" src="images/2-4-open-recognition.png" class="business-partners-logo">
+        </div>
+        <div class="business-partners-display">
+            <img alt="Bruxelles Formation logo" src="images/2-5-Brussels.png" class="business-partners-logo">
+        </div>
```

Comment:
instead of tiny little div's they all should be in a container like div or section.  so it is easier and more readable if section ends here not above

Reply:
@erkamguresen 
I tottaly agree that putting the list of immages under a container div section instead of opening and closing div's for each and every image.
Cause,It is more clear to read the code but also it saves time.

However, unfortunately i found some problems with it.
I did exactly the same but the result i got is shown below in the screen shoot.
It just lists all the logos vertically instead of horizonatally.

kindly, If one of you can give me some solutions for that (to grid the images horizontaly without using div's for each) then may be i can do it again. In my part i tried to make some ammendments like **flooting** & ** alignment** in the scc/setyle section but couldn't solve it. 🔒 

![image](https://user-images.githubusercontent.com/75224973/108607856-4649f100-73c3-11eb-8734-257de4e45060.png)

Reply:
hi @Mika215  I find very useful the following video tutorials in this modules suggested study 
1. loruki web site tutorial itself
2. FCC, Responsive Web Design:  [video](https://www.youtube.com/watch?v=srvUrASNj0s)

If you look the original website they used two main div tag. first for h2 and a tag second for images.
![partner section](https://user-images.githubusercontent.com/46300684/108608724-1dc4f580-73c9-11eb-86a3-61a86522a6d0.png)

so  you can use flex in the main div that contains all small div tags containing each images here is the view. I added border for each item with different color to see which element is where:

![127 0 0 1_5500_snippet html](https://user-images.githubusercontent.com/46300684/108609548-dc374900-73ce-11eb-895b-ccfa7b7c67ba.png)

code for html & css you can try on your computer:
```html
<!--partners section  -->
    <section class="partners-section">
      <!-- text part -->
      <div class="partners-text">
        <div>
          <h2>Our Partners</h2>
        </div>

        <!--Become a Partner-->
        <div>
          <a href="mailto:contact@hackyourfuture.be">Become a Partner</a>
        </div>
      </div>

      <!-- logo part -->
      <div class="partners-logos">
        <div>
          <img
            alt="PrintSimple logo"
            src="/images/1-1-print-smple.svg"
            class="business-partners-logo"
          />
        </div>

        <div>
          <img
            alt="Open Knowledge Belgium logo"
            src="/images/1-2-open%20-knowledge.svg"
            class="business-partners-logo"
          />
        </div>

        <div>
          <img
            alt="Digital Belgium logo"
            src="/images/1-3-digital-belgium.svg"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Artevelde Hoge School logo"
            src="/images/1-4-artevelde.svg"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Bruxelles Economie et Emploi logo"
            src="/images/1-5-bruxelles.svg"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Microsoft logo"
            src="/images/1-6-Microsoft.svg"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Duo for Job logo"
            src="/images/1-7-Duo .png"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Member logo png"
            src="/images/2-1-member.png"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Data Camp logo"
            src="/images/2-2-datacamp.svg"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="UX Antwerp logo"
            src="/images/2-3-Antwerp.png"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Open Recognition logo"
            src="/images/2-4-open-recognition.png"
            class="business-partners-logo"
          />
        </div>

        <div class="business-partners-display">
          <img
            alt="Bruxelles Formation logo"
            src="/images/2-5-Brussels.png"
            class="business-partners-logo"
          />
        </div>
      </div>
    </section>
```
```css
.partners-section {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  border: 1px solid yellow;
  width: 95%;
  margin: 0 auto;
}

.partners-text {
  display: flex;
  justify-content: space-between;
  align-items: center;
  border: 1px solid greenyellow;
  /* width: 95%; */
  margin: 2px 2px;
}

.partners-logos {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
}

.partners-logos img {
  border: 1px solid orange;
  max-width: 180px;
  margin: 2px 2px;
}
```

Reply:
For your information, `display: flex` applies to _all immediate children_ of the container tag. So these children can be `<img>` tags, they don't need to be `<div>` tags to work.

In my opinion, your `<div>` tags around images are unnecessary, and add complexity to your source code without benefits.

(The HackYourFuture.be source code shows a lot of `<div>` tags because the HTML has been generated by a tool, not manually coded by a human. It's not really the best example code for good practices, semantics, accessibility, and readability.)